### PR TITLE
google_sign_in_web Updating README.md

### PR DESCRIPTION
**Pull request by owersbrett**
## Description

Grammatical error in google_sign_in_web

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy]. 
- [ ] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/google_sign_in/google_sign_in_web/README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ See the [google_sign_in.dart](https://github.com/flutter/plugins/blob/master/pac
 
 ## Contributions and Testing
 
-Tests are a crucial to contributions to this package. All new contributions should be reasonably tested.
+Tests are crucial to perform before contributing to this package. All new contributions should be reasonably tested.
```
Comment (stuartmorgan) [inline packages/google_sign_in/google_sign_in_web/README.md:99]: This is changing the meaning; the intent is not just that people need to run tests, but also add tests.

Just removing "a" would fix the grammar without changing the meaning.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks for the contribution! We apologize for the long delay in triaging this PR. We’re in the process of overhauling our PR triage system to respond much more quickly, as well as working through the backlog.

This will need updates to the version in the pubspec and a corresponding addition to the changelog, since without those we can't publish the README update.

**Comment by owersbrett** [comment]
I see! So I would say that I think the word we are looking for is
essential. "Tests are essential to contributions to this package", or
"Tests are crucial components to contributions to this package". Crucial is
an adjective, whereas essential can be used as both an adjective and a
noun. Should "crucial" be modifying a word "tests are a crucial component
to every contribution to this package", as opposed to being referred to as
the actual component of a noun "a crucial to contributions", the grammar
would be correct, but I think the word we're looking for is essential. I
know it seems small, but I had made this PR when I first started learning
Flutter and I wanted to get involved in the open source community, and
thought what better way to start then helping to improve some documentation
if I can. I appreciate you getting back to me, have a great day!

**Comment by stuartmorgan** [comment]
> "Tests are essential to contributions to this package" [...]
> Crucial is an adjective, whereas essential can be used as both an adjective and a noun.

Essential isn't a noun in your proposed sentence, so I'm not sure how that's relevant.

There's a minor grammatical quibble as to whether it would be "critical/essential/crucial/etc *to*" or "... *for*" in this sentence, but the distinction you are drawing between "essential" and "critical" in this construction doesn't exist.

**Comment by stuartmorgan** [comment]
Obsoleted by #3816, so closing. Thanks again for flagging this typo!

**Pull request closed without merging**